app/models/venue.rb

```diff
@@ -1,6 +1,5 @@
 class Venue < ApplicationRecord
   belongs_to :user
-  has_many :reviews
```

Comment:
Does this get replaced by the line 'has_many :reviews, through: :venue' in user model?

Reply:
venue model should have this:
belongs_to :user
  has_many :reviews, dependent: :destroy
  has_many :bookings, dependent: :destroy

I did some test during flight so will cancel this change.